src/translations/pl/translation.json

```diff
@@ -444,46 +444,46 @@
-    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_other": "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}}": "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}}",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_one": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_few": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_many": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_other": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}}": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}}",
+    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_one": "Mając {{effectNames, list}} efekt(-y) na twojej {{bodyParts, list(type: disjunction)}} na {{operator}} {{count}} sekund",
+    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_few": "Mając {{effectNames, list}} efekt(-y) na twojej {{bodyParts, list(type: disjunction)}} na {{operator}} {{count}} sekund",
+    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_many": "Mając {{effectNames, list}} efekt(-y) na twojej {{bodyParts, list(type: disjunction)}} na {{operator}} {{count}} sekund",
+    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_other": "Mając {{effectNames, list}} efekt(-y) na twojej {{bodyParts, list(type: disjunction)}} na {{operator}} {{count}} sekund",
```

Comment:
I think `seconds_other` il `pl` applies when seconds are not a number but a word, like _few_ or _many_, so maybe it should be `sekundy`? But I'm not really sure 🤔

Eg:

> While having the **pain** effect(s) on your **arm** for **several** seconds

p.s. this comment applies to other strings too where you see the seconds_few quantifier